KubeArmor/build/compile.sh

```diff
@@ -12,3 +12,4 @@ else
 fi
 
 cp *.bpf.o /opt/kubearmor/BPF/
+cp *.bpf.o ignore.lst /opt/kubearmor/BPF/
```

Comment:
Few points:
1. `*.bpf.o` is getting copied in two places.
2. What is the reason to copy `ignore.lst` ?

Reply:
we are generating `ignore.lst` based on syscheck result and kubearmor will require this info while loading system monitor which is missing in current kubearmor images and resulting in failing for the environment with unavailable hooks i.e. on rhel9 5.14.0 lsm hooks for unlink,rmdir is unavailable.

Reply:
@achrefbensaad , can you check if this is inline to your understanding? I thought we generate `ignore.lst` dynamically in the init container.

Reply:
Yes we generate ignore.lst dynamically in the init container, but we need to copy it over to the main container so that KubeArmor knows what hooks it should not load.